extensions/amp-ad/amp-ad.md

```diff
@@ -452,7 +452,7 @@ See [amp-ad rules](validator-amp-ad.protoascii) in the AMP validator specificati
 -   [Sklik](../../ads/vendors/sklik.md)
 -   [SSP](../../ads/vendors/ssp.md)
 -   [SlimCut Media](../../ads/vendors/slimcutmedia.md)
--   [Smart AdServer](../amp-ad-network-smartadserver-impl/readme.md)
+-   [Smart AdServer](../amp-ad-network-smartadserver-impl/amp-ad-network-smartadserver-impl-internal.md)
```

Comment:
Can you do this in a diff PR?

Reply:
https://github.com/ampproject/amphtml/pull/37566